### PR TITLE
Ebean 13.6.0 compatibility

### LINKED ISSUE #278: NoClassDefFoundError: io/ebean/EbeanServer
I updated Ebean to 13.6.0, and got this exception:
```
play.api.UnexpectedException: Unexpected exception[NoClassDefFoundError: io/ebean/EbeanServer]
	at play.core.server.DevServerStart$$anon$1.reload(DevServerStart.scala:258)
	at play.core.server.DevServerStart$$anon$1.get(DevServerStart.scala:148)
	at play.core.server.AkkaHttpServer.handleRequest(AkkaHttpServer.scala:302)
	at play.core.server.AkkaHttpServer.$anonfun$createServerBinding$1(AkkaHttpServer.scala:224)
	at akka.stream.impl.fusing.MapAsync$$anon$30.onPush(Ops.scala:1307)
	at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:542)
	at akka.stream.impl.fusing.GraphInterpreter.processEvent(GraphInterpreter.scala:496)
	at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:390)
	at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:650)
	at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:521)
Caused by: java.lang.NoClassDefFoundError: io/ebean/EbeanServer
	at java.base/java.lang.Class.getDeclaredMethods0(Native Method)
	at java.base/java.lang.Class.privateGetDeclaredMethods(Class.java:3402)
	at java.base/java.lang.Class.getDeclaredMethods(Class.java:2504)
	at com.google.inject.internal.DeclaredMembers.getDeclaredMethods(DeclaredMembers.java:48)
	at com.google.inject.spi.InjectionPoint.getDeclaredMethods(InjectionPoint.java:811)
	at com.google.inject.spi.InjectionPoint.getInjectionPoints(InjectionPoint.java:730)
	at com.google.inject.spi.InjectionPoint.forInstanceMethodsAndFields(InjectionPoint.java:430)
	at com.google.inject.internal.ConstructorBindingImpl.getInternalDependencies(ConstructorBindingImpl.java:177)
	at com.google.inject.internal.InjectorImpl.getInternalDependencies(InjectorImpl.java:670)
	at com.google.inject.internal.InjectorImpl.cleanup(InjectorImpl.java:627)
Caused by: java.lang.ClassNotFoundException: io.ebean.EbeanServer
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:445)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:587)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at java.base/java.lang.Class.getDeclaredMethods0(Native Method)
	at java.base/java.lang.Class.privateGetDeclaredMethods(Class.java:3402)
	at java.base/java.lang.Class.getDeclaredMethods(Class.java:2504)
	at com.google.inject.internal.DeclaredMembers.getDeclaredMethods(DeclaredMembers.java:48)
	at com.google.inject.spi.InjectionPoint.getDeclaredMethods(InjectionPoint.java:811)
	at com.google.inject.spi.InjectionPoint.getInjectionPoints(InjectionPoint.java:730)
	at com.google.inject.spi.InjectionPoint.forInstanceMethodsAndFields(InjectionPoint.java:430)
```
Links:
https://github.com/ebean-orm/ebean/releases/tag/ebean-parent-13.6.0
https://github.com/ebean-orm/ebean/pull/2662

### PR DESCRIPTION
**Pull request by PromanSEW**
Migration from deprecated classes
Fixes #278
Closes #277
Closes #280

### CODE REVIEW

**Comment by PromanSEW** [comment]
`README` must be updated that `6.2.0-RC4` supports Ebean up to 13.5.0

**Comment by PromanSEW** [comment]
@mkurz please review

**Comment by mkurz** [comment]
> @mkurz please review

Doing right now, I can release a new RC as well.

**Comment by PromanSEW** [comment]
I missed that Ebean 13 does not support Java 8...

**Comment by mkurz** [comment]
Yes, please downgrade again to the version used in the main branch now.

**Comment by mkurz** [comment]
You can upgrade to the newest version yourself anyway.

**Comment by mkurz** [comment]
See this comment for Java 11: https://github.com/playframework/play-ebean/pull/252#issuecomment-1084208079

File: play-ebean/src/main/java/play/db/ebean/EbeanDynamicEvolutions.java [hunk 1]
```diff
@@ -65,15 +66,11 @@ public void create() {
             }
             File evolutions = environment.getFile("conf/evolutions/" + key + "/1.sql");
             try {
-                String content = "";
-                if (evolutions.exists()) {
-                    content = new String(Files.readAllBytes(evolutions.toPath()), "utf-8");
-                }
-
+                String content = evolutions.exists() ? Files.readString(evolutions.toPath()) : "";
```
Comment (PromanSEW) [inline play-ebean/src/main/java/play/db/ebean/EbeanDynamicEvolutions.java]: IDEA still suggests replace with non-existent method after changing JDK to 1.8. Nice bug

**Comment by PromanSEW** [comment]
Ping @mkurz

**Comment by mkurz** [comment]
Will take a look today and release if everything is ok.

**Review by mkurz — APPROVED** [review]
LGTM, thanks!

**Comment by mkurz** [comment]
@PromanSEW https://github.com/playframework/play-ebean/releases/tag/6.2.0-RC5

**Pull request merged**